src/soroban_rpc.ts

```diff
@@ -131,7 +131,7 @@ export namespace SorobanRpc {
     transactionData: string;
     events: string[];
     minResourceFee: string;
-    results: SimulateHostFunctionResult[];
+    results: SimulateHostFunctionResult[]; // always one element, tho
```

Comment:
```suggestion
    // This will only contain a single element, because only a single invokeHostFunctionOperation is supported per transaction.
    results: SimulateHostFunctionResult[];
```